Description: Correct POD issues in LedgerSMB/Form.pm
 Has the incomplete module name but not have the dash plus short description in
 the NAME entry.
 Spelling error in lines 2091,2397,2960,3104,3139, & 3143: seperated separated.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472015&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Form.pm
+++ b/LedgerSMB/Form.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-Form
+LedgerSMB::Form - Provides general legacy support functions and the central object
 
 =head1 SYNOPSIS
 
@@ -2088,7 +2088,7 @@
 =item $form->all_taxaccounts($myconfig, $dbh2[, $transdate]);
 
 Get the tax rates and numbers for all the taxes in $form->{taxaccounts}.  Does
-nothing if $form->{taxaccounts} is false.  Taxes are listed as a space seperated
+nothing if $form->{taxaccounts} is false.  Taxes are listed as a space separated
 list of account numbers from the chart.  The retrieved values are placed within
 $form->{${accno}_rate} and $form->{${accno}_taxnumber}.  If $transdate is set,
 then only process taxes that were valid on $transdate.
@@ -2394,7 +2394,7 @@
 accounts that have $module in their link field.  The hash is keyed upon link
 elements such as 'AP_amount' and 'AR_tax' and they refer to lists of hashes
 containing accno and description for the appropriate accounts.  If the key does
-not contain 'tax', the account number is appended to the space seperated list
+not contain 'tax', the account number is appended to the space separated list
 $form->{accounts}.  $module is typically 'AR' or 'AP' and is the base type of
 the accounts looked up.
 
@@ -2957,7 +2957,7 @@
 Clears out any old status entries for $form->{id} and saves new status entries.
 Queued form names are extracted from $form->{queued}.  Printed and emailed form
 names are extracted from $form->{printed} and $form->{emailed}.  The queued,
-printed, and emailed fields are space seperated lists.
+printed, and emailed fields are space separated lists.
 
 =cut
 
@@ -3101,7 +3101,7 @@
 Saves or deletes recurring transaction scheduling.  $form->{id} is used to
 determine the id used in the various recurring tables.  A recurring transaction
 schedule is deleted by having $form->{recurring} be false.  For adding or
-updating a schedule, $form->{recurring} is a comma seperated field with partial
+updating a schedule, $form->{recurring} is a comma separated field with partial
 subfield quoting of the form:
 
   reference,startdate,repeat,unit,howmany,payment,print,email,message
@@ -3136,11 +3136,11 @@
 
 =item print
 
-A colon seperated list of formname:format:printer triplets.
+A colon separated list of formname:format:printer triplets.
 
 =item email
 
-A colon seperated list of formname:format pairs.
+A colon separated list of formname:format pairs.
 
 =item message
 
